ksqldb-tools/src/test/java/io/confluent/ksql/tools/migrations/util/CommandParserTest.java

```diff
@@ -124,10 +151,27 @@ public void shouldParseSetUnsetStatements() {
     assertTrue(!((SqlPropertyCommand) commands.get(1)).getValue().isPresent());
   }
 
+  @Test
+  public void shouldParseSetUnsetStatementsWithVariables() {
```

Comment:
nit: follow given/when/then pattern for improved readability, and consider breaking this into two tests if the fact that the two commands are parsed together is not relevant to this test.